js/stores/windowStore.js

```diff
@@ -135,14 +135,10 @@ const doAction = (action) => {
     case WindowConstants.WINDOW_SET_LOCATION:
       const key = action.key || windowState.get('activeFrameKey')
       const lastLocation = windowState.getIn(frameStatePath(key).concat(['location']))
-      const lastTitle = windowState.getIn(frameStatePath(key).concat(['title']))
-      let locationChanged = !action.location || !lastLocation ||
-        action.location.split('#')[0] !== lastLocation.split('#')[0]
       windowState = windowState.mergeIn(frameStatePath(key), {
         audioPlaybackActive: false,
         adblock: {},
         trackingProtection: {},
-        title: locationChanged ? '' : lastTitle,
```

Comment:
Actually I'm surprised if this passes the test `page without a title`, maybe that test needs a tweak if it does pass.